tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -14,6 +14,27 @@
  */
 class Tests_Theme_wpThemeJson extends WP_UnitTestCase {
 
+	/**
+	 * Administrator ID.
+	 *
+	 * @var int
+	 */
+	private static $administrator_id;
+
+	public static function set_up_before_class() {
+		parent::set_up_before_class();
+
+		static::$administrator_id = self::factory()->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+
+		if ( is_multisite() ) {
+			grant_super_admin( self::$administrator_id );
+		}
+	}
```

Comment:
@glendaviesnz the tests are failing because this needs to add a `$user_id`  also.  Notice in the `data_custom_css_for_user_caps()` data provider, it tests the admin and then tests a non-admin, i.e. `static::$user_id`.

So need to another another property and create a user.

```suggestion

	/**
	 * User ID.
	 *
	 * @var int
	 */
	private static $user_id;

	public static function set_up_before_class() {
		parent::set_up_before_class();

		static::$administrator_id = self::factory()->user->create(
			array(
				'role' => 'administrator',
			)
		);

		if ( is_multisite() ) {
			grant_super_admin( self::$administrator_id );
		}

		static::$user_id = self::factory()->user->create();
	}
```

Reply:
Doh! was there but I removed it by mistake!